<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Preferred fiat currency:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_fiatCurrency">
         <item>
          <property name="text">
           <string>USD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EUR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RUB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GBP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AUD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CAD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CHF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CNY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CZK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JPY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KRW</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MXN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NZD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SEK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>THB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TRY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ZAR</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Appearance:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_skin"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Block explorer:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_blockExplorer">
         <item>
          <property name="text">
           <string>explore.wownero.com</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blockchair.com</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_externalLink">
         <property name="text">
          <string>Warn before opening external link</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBox_hideBalance">
         <property name="text">
          <string>Hide balances</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Reddit frontend:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_redditFrontend">
         <item>
          <property name="text">
           <string>old.reddit.com</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>reddit.com</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>teddit.net</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_hideFiatBalance">
         <property name="text">
          <string>Hide fiat balance</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_node">
      <attribute name="title">
       <string>Node</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="NodeWidget" name="nodeWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_network">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Websockets</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_disableWebsocket">
              <property name="text">
               <string>Disable websocket</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_websocketServer">
              <property name="text">
               <string>Websocket server</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit_websocketServer">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_paths">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>This application uses the following paths:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineEdit_defaultWalletDir">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_browseDefaultWalletDir">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Wallet directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Config directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_configDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Application directory:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_applicationDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_console">
      <attribute name="icon">
       <iconset theme=":/assets/images/terminal.png">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Console</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="title">
          <string>Log source</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkBox_12">
            <property name="text">
             <string>Qt Application</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_13">
            <property name="text">
             <string>RPC daemon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_14">
            <property name="text">
             <string>Tor daemon</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnCopyToClipboard">
            <property name="text">
             <string>Copy to clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Open log directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="title">
          <string>Log level</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="checkBox_9">
            <property name="text">
             <string>Critical</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_10">
            <property name="text">
             <string>Warning</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_8">
            <property name="text">
             <string>Info</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_11">
            <property name="text">
             <string>Debug</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="textLogs">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="closeButton">
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NodeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/nodewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>copyToClipboard()</slot>
  <slot>checkboxExternalLinkWarn()</slot>
  <slot>closeClicked(QAbstractButton*)</slot>
  <slot>checkboxCheckForAppUpdates()</slot>
  <slot>fiatCurrencySelected(QString)</slot>
  <slot>homeWidgetChanged(int)</slot>
 </slots>
</ui>
